civihr_employee_portal/src/Mail/OnboardingFormEditNotificationMail.php

```diff
@@ -0,0 +1,157 @@
+   */
+  protected function formatSubmittedValues($node, $submission) {
+    $submittedValues = WebformHelper::getSubmittedValues($node, $submission);
+
+    $this->formatPersonalDetails($submittedValues);
+    $this->formatPayroll($submittedValues);
+    $this->formatEmergencyContacts($submittedValues);
+    $this->formatDependants($submittedValues);
+
+    // We don't want this info in the email
+    unset($submittedValues['Profile Picture']);
+
+    return $submittedValues;
+  }
+
+  /**
+   * @param array $submittedValues
+   */
+  private function formatPersonalDetails(&$submittedValues) {
+    $personalDetails = &$submittedValues['Personal Details'];
+    unset($personalDetails[0]['Existing Contact']);
+
+    // The options for this are not stored correctly and need manual replacement
+    $ssnApplicationKey = 'I am currently applying for a NI/ SSN';
+    $value = $personalDetails[0][$ssnApplicationKey];
+    $replacement = $value ? 'Yes' : 'No';
+    $personalDetails[0][$ssnApplicationKey] = $replacement;
+  }
+
+  /**
+   * @param $submittedValues
+   */
+  protected function formatPayroll(&$submittedValues) {
+    $payroll = &$submittedValues['Payroll'];
+    $skipStepOption = 'Add Now (Payroll)';
+    if ($this->skippedStep($payroll, $skipStepOption)) {
+      $this->formatSkippedStep($payroll);
+    }
+    unset($payroll[0][$skipStepOption]);
+  }
+
+  /**
+   * @param $submittedValues
+   */
+  private function formatEmergencyContacts(&$submittedValues) {
+    $emergencyContacts = &$submittedValues['Emergency Contact'];
+    $skipStepOption = 'Add Now (Emergency Contacts)';
+
+    if ($this->skippedStep($emergencyContacts, $skipStepOption)) {
+      $this->formatSkippedStep($emergencyContacts);
+      return;
+    }
+    unset($emergencyContacts[0][$skipStepOption]);
+
+    $addSecondOption = 'Add another emergency contact?';
+    $firstContactKey = 'Emergency Contact';
+    $secondContactKey = 'Second Emergency Contact';
+
+    unset($emergencyContacts[$firstContactKey]['Is a dependant?']);
+    unset($emergencyContacts[$secondContactKey]['Is a dependant?']);
+
+    // Check if a second contact was added and remove the value for conditional
+    $addedSecondContact = $emergencyContacts[0][$addSecondOption] === 'Yes';
+    unset($emergencyContacts[0][$addSecondOption]);
+
+    // Remove the second emergency contact if it wasn't added
+    if (!$addedSecondContact) {
+      unset($emergencyContacts[$secondContactKey]);
+    }
+  }
+
+  /**
+   * @param array $submittedValues
+   */
+  private function formatDependants(&$submittedValues) {
+    $dependants = &$submittedValues['Dependants'];
+    $skipStepOption = 'Add Now (Dependants)';
+    if ($this->skippedStep($dependants, $skipStepOption)) {
+      $this->formatSkippedStep($dependants);
+      return;
+    }
+    // Unset unused data
+    unset($dependants[0]);
+    // Remove unused dependant entries
+    $dependantsKeys = ['Fifth', 'Fourth', 'Third', 'Second', 'First'];
```

Comment:
Do you mind renaming this to `$dependantKeys` since you are using `$dependantKey` in the foreach. 🤔 
No errors just common convention.